### ISSUE TITLE
FYI: Thanks

### ISSUE THREAD

**Issue by alphapapa** [m00natic/cl-fdbq#1]
Hi,

Just wanted to say thanks for posting this and the accompanying article.  It is like a more advanced version of what I'm doing in https://github.com/alphapapa/org-ql.  Of course, I can't do in Emacs Lisp everything that CL can do, but maybe I can learn from your examples and apply some things to org-ql.  :)

**Comment by m00natic** [comment]
Thank you for the heads up!

I use org regularly but not much it's agenda and TODO features.  One of
the subconscious reasons is that pouring data over data makes it a bit
hard to disentangle quickly.  The other reason is that I'm too lazy to
write that much ¯\_(ツ)_/¯ Anyway, org-ql looks a cool way to handle the
first problem.  Good work, sir!

Emacs Lisp is probably today's closest dialect to CL, has the CL
emulation layer and is no less fun but has its limitations especially on
the performance side.

The article is missing a few final touches but soon will be declared
ready.  We'll see where cl-fdbq goes and if it can become a practical
and reusable mechanism.

**Comment by alphapapa** [comment]
> Emacs Lisp is probably today's closest dialect to CL, has the CL emulation layer and is no less fun but has its limitations especially on the performance side.

Indeed.  I'm hoping the Elisp JIT that's been discussed recently pans out!  And one of these days maybe I'll work on parsing Org in CL, although that's probably over my head.  TXR Lisp also looks very interesting for parsing.
